tests/Conference/ConferenceHarvesterTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Tests\Conference;
+
+use App\Conferences\ConferencesHarvester;
+use App\Entity\Conference;
+use App\Entity\ConferenceFilter;
+use App\Entity\FetcherConfiguration;
+use App\Fetcher\TululaFetcher;
+use App\Repository\ConferenceFilterRepository;
+use App\Repository\ConferenceRepository;
+use App\Repository\FetcherConfigurationRepository;
+use Doctrine\ORM\EntityManager;
+use Prophecy\Argument;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class ConferenceHarvesterTest extends KernelTestCase
+{
+    /**
+     * @dataProvider provideFetcherResponses
+     */
+    public function testFetch(array $fetcherData)
```

Comment:
$fetcherData is a big black box. I would be better to use something like:

```suggestion
    public function testFetch(int $expectedUpdated, int $expectedAdded, string $name, bool isActive, XXXXX)
```

* Easier to read
* Benefit of type hint